.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     parameters:
       ruby_version:
         type: string
-        default: 2.7.8
+        default: 3.2.2
```

Comment:
I believe Kirk has a commit for this file? If not, I think we may need to update the rails version too.

Reply:
He did not. I will update rails version as well.